examples/sample-app/application-template-stibuild.json

```diff
@@ -39,6 +39,9 @@
       "kind": "ImageRepository",
       "metadata": {
         "name": "ruby-20-centos7"
+      },
+      "tags": {
+        "latest": "latest"
```

Comment:
you should never need to do this.  either the tag will be created because you did a push to the docker registry, or the tag does not exist because the image does not exist.  you should never directly create the tag.